app/controllers/application_controller.rb

```diff
@@ -4,4 +4,8 @@ class ApplicationController < ActionController::Base
   helper_method :admin?
 
   delegate :admin?, to: :current_user, allow_nil: true
+
+  def not_found
+    render file: "#{Rails.root}/public/404", layout: false, status: :not_found
```

Comment:
@kushniryb @denysmedynskyi I am not sure that this is the best way to response with 404... what do you think guys ?